crypto-macros/src/entity_derive/parse.rs

```diff
@@ -82,47 +82,52 @@ impl IdColumn {
                 .expect("named fields always have identifiers")
                 .clone();
 
-            if let Some(attr) = field.attrs.iter().find(|a| a.path().is_ident("id")) {
-                let mut column_name = None;
-                let mut transformation = None;
-                let column_type = IdColumnType::parse(&field.ty)?;
-
-                if let Ok(list) = attr.meta.require_list() {
-                    list.parse_nested_meta(|meta| {
-                        match meta.path.require_ident()?.to_string().as_str() {
-                            "column" => {
-                                meta.input.parse::<Token![=]>()?;
-                                column_name = Some(meta.input.parse::<syn::LitStr>()?.value());
+            match field.attrs.iter().find(|a| a.path().is_ident("id")) {
+                Some(attr) => {
+                    let mut column_name = None;
+                    let mut transformation = None;
+                    let column_type = IdColumnType::parse(&field.ty)?;
+
+                    if let Ok(list) = attr.meta.require_list() {
+                        list.parse_nested_meta(|meta| {
+                            match meta.path.require_ident()?.to_string().as_str() {
+                                "column" => {
+                                    meta.input.parse::<Token![=]>()?;
+                                    column_name = Some(meta.input.parse::<syn::LitStr>()?.value());
+                                }
+                                "hex" => transformation = Some(IdTransformation::Hex),
+                                _ => return Err(syn::Error::new_spanned(meta.path, "unknown argument")),
```

Comment:
I assume this is the result of `cargo fix` -- is this really necessary in our particular case?

Reply:
No, and fixed it.